### DISCUSSION TITLE
Evaluating functions with one argument parenthesis unnecessary

### DISCUSSION THREAD

**Discussion by kubikas3** [josdejong/mathjs#2298] [Ideas]
Is it possible to add feature to make parenthesis uncesessary for functions with one argument? (similar to implicit multiplication)
For example:

```js
math.evaluate('sin x'); // instead of sin(x)
math.evaluate('sin -x');
math.evaluate('sin x^2');
math.evaluate('sqrt 9 + sqrt 9'); // = 6
...
```

**Comment by josdejong** [comment]
I'm afraid not: this conflicts with implicit multiplication, which interprets `sin x` as `sin * x`.

**Reply by gwhitney** [reply]
I mean, humans interpret `sin x` correctly, because sin is a symbol for a function. So conceivably there could be such a feature, but it might make the mathjs language unparseable or nearly so. But anyhow, moving this to "Ideas", even if it's a long shot that this could be implemented.
